src/mjolnir/adminbuilder.cc

```diff
@@ -645,45 +645,44 @@ bool BuildAdminFromPBF(const boost::property_tree::ptree& pt,
   }
   LOG_INFO("Created Level index");
 
-  sql = "CREATE INDEX IdxDriveOnRight ON admins (drive_on_right)";
+  sql = "UPDATE admins AS child SET parent_admin = (SELECT parent.rowid from admins";
```

Comment:
The diff is horribly incomprehensible, but it's simple: do the `parent_admin` update first, then `drive_on_right`, then `allow_intersection_names`.